pkg/api/types.go

```diff
@@ -1,5 +1,10 @@
 package api
 
+const (
+	// APIVersion is the version of this API
+	APIVersion = "master"
```

Comment:
we've use "unstable" in the past.  I don't have a preference.

Reply:
@asalkeld what about "internal" ?

Reply:
ok going with internal.